ds_judgements_public_ui/templates/pages/transactional_licence.html

```diff
@@ -6,7 +6,9 @@ <h2>Part 1 &ndash; Instructions</h2>
   <h3>How to apply</h3>
   <ol>
     <li>Read Part 1 &ndash; Instructions.</li>
-    <li>Answer all the relevant questions in Part 2 &ndash; Application Form.</li>
+    <li>
+      Answer all the relevant questions in Part 2 &ndash;<a href="https://caselaw.nationalarchives.gov.uk/static/documents/application_form_for_licence_to_re-use.docx"> Application Form.</a>
```

Comment:
I'm tempted to say this should be an anchor link to Part 2, rather than a link to the Word file directly?

Reply:
This looks something like: 
<a href='#part2'>Application Form</a>
(then later)
<h2 id='part2'>Part 2</h2>